CHANGELOG.md

```diff
@@ -32,6 +32,7 @@
 
 - **General:** Automatically release container image for ARM ([#2263]https://github.com/kedacore/keda/issues/2263))
 - **General:** Automatically run end-to-end tests on ARM ([#2262]https://github.com/kedacore/keda/issues/2262))
+- **General:** Provide support for authentication via Azure Key Vault  ([#900](https://github.com/kedacore/keda/issues/900))
```

Comment:
Sorry for being pain in the ***, but could you please rebase and keep alphabetical order here? :)